### ISSUE TITLE
OpenComputers monitor glitches out when using remote, and then does not work until you break the block

### ISSUE THREAD

**Issue by prozacgod** [orbwoi/UniversalRemote#24]
I can open the GUI on this block, but it permanently stops the block from working until after you break it/replace it

Keyboard commands seem to get through, but the screen can no longer draw updates.

**Comment by orbwoi** [comment]
Yeah, both computercraft and open computers don't use the normal GUI functions. They are currently not supported and will probably not work correctly. I suppose I could blacklist them for now to prevent confusion?
